### ISSUE TITLE
Provide way to pass connection parameters like sock conn timeout etc in client connection

### ISSUE THREAD

**Issue by skyrocknroll** [ybrs/pydisque#28]
Right now client.connect() does not take any redis client connection paramaters. We should be able to pass all the redis config parameters like tcp sock connect time out etc..

**Comment by canardleteer** [comment]
This is a good idea, probably just a way to pass along anything to redis.Redis.

**Comment by ybrs** [comment]
fixed at ea5ce1576b66398c1cce32cad0f15709b1ea8df8

one can pass kwargs to Client - which will pass to redis.Redis directly 

eg:
```
Client(['localhost:7712', 'localhost:7711'], password='foobared')
```
